### PR TITLE
feat: usd wallet creation script

### PR DESCRIPTION
**Pull request by krtk6160**

### CODE REVIEW

**Comment by nicolasburtey** [comment]
this one can be closed?

**Comment by krtk6160** [comment]
> this one can be closed?

Yes

**Comment by vindard** [comment]
**Adding context for why this is still open**

- We now have an admin mutation that can bulk add usd wallets to accounts (see #1380)
- We would still need to run a one-time migration for _certain_ environments to add usd wallet to _all_ users at some point
- This is being done as a debug script instead of migration script because we likely would not want to apply USD wallets to all current environments that we manage
- We will likely always need this script if we assume that a new client/environment can start as BTC-only and later transition to include USD wallets

**Review by vindard — APPROVED** [review]
Works well on testing, on thing is the debug process doesn't terminate locally when I run with `yarn ts-node --files -r tsconfig-paths/register src/debug/create-usd-wallets.ts` but this is the same with other similar debug scripts so it's likely ok.

File: src/debug/create-usd-wallets.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import { setupMongoConnection } from "@services/mongodb"
+import { AccountsRepository } from "@services/mongoose"
+
+const createUsdWallets = async () => {
+  await setupMongoConnection()
+
+  const accounts = await AccountsRepository().listUnlockedAccounts()
+  if (accounts instanceof Error) return accounts
+
+  for await (const account of accounts) {
```
[the remark this answers is not in the document]
Comment (krtk6160) [reply src/debug/create-usd-wallets.ts]: Fixed

**Pull request merged**